bindings/python/test.py

```diff
@@ -75,6 +76,18 @@ def test_torrent_status(self):
 		st2 = self.h.status()
 		self.assertEqual(st2, st)
 
+	def test_status_last_uploaded_dowloaded(self):
+		# we want to check at seconds precision but can't control session
+		# time, wait for next full second to prevent second increment
+		time.sleep(1 - datetime.datetime.now().microsecond / 1000000.0)
+
+		sessionStart = datetime.datetime.now().replace(microsecond=0)
```

Comment:
@arvidn is that ok as it is or should I change it to assert(diff < seconds(2)) (2 = is max session start time 1s + 1s safety for secound wrap)  to be able to remove the sleep